anvil/src/eth/backend/fork.rs

```diff
@@ -142,6 +147,20 @@ impl ClientFork {
         self.provider().fee_history(block_count, newest_block, reward_percentiles).await
     }
 
+    /// Sends `eth_getProof`
+    pub async fn get_proof(
+        &self,
+        address: Address,
+        keys: Vec<U256>,
+        block_number: Option<BlockId>,
+    ) -> Result<AccountProof, ProviderError> {
+        let address = ethers::utils::serialize(&address);
+        let keys = ethers::utils::serialize(&keys);
+        let block_number =
+            ethers::utils::serialize(&block_number.unwrap_or_else(|| BlockNumber::Latest.into()));
+        self.provider().request("eth_getProof", [address, keys, block_number]).await
```

Comment:
https://github.com/gakonst/ethers-rs/blob/master/ethers-providers/src/provider.rs#L814 FYI we also have this

Reply:
oh nice, can unify the responses as well